src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import COLORS from '../styles/colors';
+
+export const Button = styled.button<{ $secondary?: boolean }>`
```

Comment:
A better name may be `$color`? I don't think `$secondary` clearly states the purpose of the prop

Reply:
Named the prop `$secondary` based on the styled-components docs example, the Figma also splits each button type (regular and error) into 'primary' and 'secondary'. I think naming it `$color` can be more confusing in my opinion, but I'm open to changing it. @pragyakallanagoudar thoughts on this??

Reply:
ohh I see, sorry I didn't read too much into what `$secondary` does. I still think we should allow for customizations to the background and text colors still (with hover color computed using helpers), which will open up the door for colors other than "secondary" without needing to copy paste this button component. Instead, we can make preset React components that fill in the values for us (i.e. `PrimaryButton`, `SecondaryButton`, `DangerButton` (for the reset password one).

Reply:
ahhh I'm just now understanding this point LOL sorry Jinkang I think I was misinterpreting this at worksession

this is good solution! but again I'll leave it up to you @varortz, i don't have a strong opinion either way

Reply:
Yeah, I already started implementing the button using jinkang's suggestion since it's good practice. Ty @jinkang-0 for the suggestion and clarifications!